test/simple.spec.coffee

```diff
@@ -270,6 +270,18 @@ define ['chai', 'cs!./test-config'], ({assert, expect}, {client, USERNAME, TOKEN
         itIsOk(REPO, 'git.trees', 'c18ba7dc333132c035a980153eb520db6e813d57', 'fetch')
         # itIsOk(REPO, 'git.trees.create', {tree: [sha], base_tree: sha})
 
+        it '.git.refs.tags.fetch()', () ->
+          STATE[GH].repos('philschatz', 'octokat.js').git.refs.tags.fetch().then (tags) ->
```

Comment:
Some simple tests, wasn't sure what do do about mocking here. The rest of the tests here seem to use a test repo, that isn't online at elast. So decided to use this repo.